docs/docs/models/ESM-2/pre-training.md

```diff
@@ -9,9 +9,7 @@ and train/test splits are available.
 
 Validation perplexity evaluated on the NVIDIA validation set.
 
-<figure markdown="span">
-  ![ESM-2 Pre-training Convergence](../assets/images/esm2/esm2_pretrain_convergence.svg){ width="350" }
-</figure>
+![ESM-2 Pre-training Convergence](../../assets/images/esm2/esm2_pretrain_convergence.png)
```

Comment:
can you check line 22? and respective ones for other model sizes? there seems to be a str left  `=== "8M"`

Reply:
Right, these are for content tabs in material mkdocs: 
https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#anchor-links-open-me-in-a-new-tab-